src/extension.ts

```diff
@@ -1083,6 +1083,14 @@ class ExtensionManager implements vscode.Disposable {
 let _EXT_MANAGER: ExtensionManager|null = null;
 let cmakeTaskProvider: vscode.Disposable | undefined;
 
+export async function registerTaskProvider(command: string | null) {
+  if (command) {
+    rollbar.invokeAsync(localize('registerTaskProvider', 'Register the task provider.'), async () => {
+      cmakeTaskProvider = vscode.tasks.registerTaskProvider(CMakeTaskProvider.CMakeType, new CMakeTaskProvider({ build: command }));
```

Comment:
I think this should check if one is already provided and dispose the old one. Now that I think of it, I think this is already broken for multi-root today since there can be multiple cmake drivers alive. Can you open a bug? For now, let's just `dispose` the existing one and replace it with a new one.